modules/core/perf/perf_precomp.hpp

```diff
@@ -5,5 +5,6 @@
 #define __OPENCV_PERF_PRECOMP_HPP__
 
 #include "opencv2/ts.hpp"
+#include "opencv2/core/softfloat.hpp"
```

Comment:
we don't need it everywhere in tests actually.

Reply:
fixed